### PR TITLE
Brand update

### PR DESCRIPTION
**Pull request by JMaio**
preview using https://jmaio.github.io/website2017/
use brand repo submodule as centralized source containing current images; changes:
- horizontal nav logo 
- homepage logo

### CODE REVIEW

**Comment by qaisjp** [comment]
I'd recommend against using a submodule as submodules are generally harder to work with

**Comment by JMaio** [comment]
(updated to not use submodule)

File: _pages/index.html [hunk 1]
```diff
@@ -7,9 +7,9 @@
   - owl.theme.default.min
 ---
 <div id="index-jumbotron" class="jumbotron text-center pt-0">
-    <img src="{{ site.baseurl }}/static/img/compsoc-profile-white-nomargin.png" class="img-fluid" alt="CompSoc Edinburgh Logo"
-    style="max-height: 300px; padding: 40px 30px">
-
+   <div style="overflow-y: hidden; max-height: 300px">
```
Comment (qaisjp) [inline _pages/index.html]: Indentation here is a little odd.

File: _pages/index.html [hunk 2]
```diff
@@ -7,9 +7,9 @@
   - owl.theme.default.min
 ---
 <div id="index-jumbotron" class="jumbotron text-center pt-0">
-    <img src="{{ site.baseurl }}/static/img/compsoc-profile-white-nomargin.png" class="img-fluid" alt="CompSoc Edinburgh Logo"
-    style="max-height: 300px; padding: 40px 30px">
-
+   <div style="overflow-y: hidden; max-height: 300px"> 
+      <img src="{{ site.baseurl }}/static/img/compsoc-profile-white-fill.png" class="img-fluid" alt="CompSoc Edinburgh Logo" style="margin-top: -40px; max-height: 400px" />
```
Comment (qaisjp) [inline _pages/index.html]: Should there be a `max-height` on the parent `div` as well as the `img` itself?

File: _pages/index.html (hunk 1 above)
Comment (JMaio) [reply _pages/index.html]: fixed!

File: _pages/index.html (hunk 2 above)
Comment (JMaio) [reply _pages/index.html]: parent div removes image padding without needing to crop it by being shorter than the image and having `overflow-y: hidden`; img has negative margin to accomplish this without losing vertical center(-ish) align

**Pull request closed without merging**